### DISCUSSION TITLE
CDN prefix in dynamic imports

### DISCUSSION THREAD

**Discussion by PKatrimoustakis** [shakacode/shakapacker#86] [Q&A]
Greetings,

Recently we've picked up splitting a couple of large modules out of the main application bundle. However I've been noticing that these modules, when prefetched, do not have a proper CDN prefix and are requested in a relative manner.

It seems like the publicPath is always relative and ignores the CDN that exists for the env. The javascript pack tags are working properly though and including the CDN prefix as expected. Anyone having a similar issue or can offer any leads into looking into this?

ENV["ASSET_HOST"] is set up properly.

Shakapacker v 6.1.1
Rails v6.1.4.7

**Comment by ahangarha** [comment]
Use `WEBPACKER_ASSET_HOST` in version 6 and `SHAKAPACKER_ASSET_HOST` in version 7+

Source: [README > Deployment](https://github.com/shakacode/shakapacker/tree/6-stable#deployment)
> If you are using a CDN setup, webpacker will use the configured [asset host](https://guides.rubyonrails.org/configuring.html#rails-general-configuration) value to prefix URLs for images or font icons which are included inside JS code or CSS. It is possible to override this value during asset compilation by setting the `WEBPACKER_ASSET_HOST` environment variable.
